README.md

```diff
@@ -81,6 +85,46 @@ tasks.cyclonedxBom {
 
 Run gradle with info logging (-i option) to see which configurations add to the BOM.
 
+__Generate BOM for multiple projects:__
+
+You can also build the BOM for multiple projects using the `--init-script` option:
+
+
+```bash
+gradle --init-script <path-to-init.gradle> cyclonedxBom -info
```

Comment:
Isn't it `--info`?

Reply:
> Isn't it `--info`?

`-info`, `--info`, and `-i` all seem to set the log level to info. I just followed [the earlier example](https://github.com/CycloneDX/cyclonedx-gradle-plugin/blob/95b35217442722f317bd2d1f8d9dfdddca92dd51/README.md?plain=1#L23) to be consistent.

Reply:
right, thanks for the answer.